core/src/main/java/io/aiven/klaw/uglify/UglifyFiles.java

```diff
@@ -23,18 +23,34 @@ public UglifyFiles() {
   private void uglifyCssFiles(String osName, Runtime rt) {
```

Comment:
Seems like we still have errors during build : https://github.com/aiven/klaw/actions/runs/4066628191/jobs/7002958112#step:5:14526

Reply:
Updated the workflow.yaml
The uglify now works although on java 19 the utility we use it is marked as deprecated. 
https://github.com/aiven/klaw/actions/runs/4072353611/jobs/7015017560#step:6:12305